README.md

```diff
@@ -53,6 +53,18 @@ engine.push(function (req, res, next, end) {
 });
 ```
 
+If you specify a `notificationHandler` when constructing the engine, JSON-RPC notifications passed to `handle()` will be handed off directly to this function without touching the middleware stack:
+
+```js
+const engine = new JsonRpcEngine({ notificationHandler });
+
+// A notification is defined as a JSON-RPC request without an `id` property.
```

Comment:
This seems like it could cause all of kinds of runtime bugs. Glad we're using TypeScript 😬